android/vpnClient/build.gradle

```diff
@@ -30,7 +30,7 @@ plugins {
 apply plugin: "org.mozilla.telemetry.glean-gradle-plugin"
 
 ext.gleanNamespace = "mozilla.telemetry.glean"
-ext.gleanYamlFiles = ["$rootDir/../../../src/telemetry/metrics_deprecated.yaml"]
```

Comment:
did you remove or rename this file?

Reply:
Yes, it was removed and just merged into the one metrics.yaml. It was useful to differentiate between Glean.js only metrics or otherwise. We don't have Glean.js anymore so 🤷